usagov/usagov-2021
# USAGOV-391-fed-tweaks: New Fed Dir Tweaks

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-391

## Description

Changed two lines to : 

• Put 25px padding below the intro paragraph, above the top of the chart
• Change the padding on the text within the chart to 25px top, left side, and bottom

Testing Steps:
May need to run  bin/gulp build if changes not reflected 
On agency detail page inspect the css and confirm that there is 25px between the chart and intro text and the padding of the text within the chart is also 25px

## Checklist for the Developer



## Checklist for the Peer Reviewers
